yarl/_url.py

```diff
@@ -272,6 +272,9 @@ def __str__(self):
     def __repr__(self):
         return "{}('{}')".format(self.__class__.__name__, str(self))
 
+    def __bytes__(self):
+        return str(self).encode("ascii")
```

Comment:
Why not UTF-8?

Reply:
It's been a while but IIRC the string representation already didn't use any non-ascii characters as everything is URL-encoded, right? Or can there be Unicode in the domain name?

Reply:
No, unicode domains are idna encoded, e.g. `xn--h1aagokeh.xn--p1ai`